ibm/service/power/data_source_ibm_pi_network_peer.go

```diff
@@ -0,0 +1,274 @@
+				Type:        schema.TypeInt,
+			},
+			Attr_CustomerCIDR: {
+				Computed:    true,
+				Description: "IP address used for configuring customer network interface with network subnet mask.",
+				Type:        schema.TypeString,
+			},
+			Attr_DefaultExportRouteFilter: {
+				Computed:    true,
+				Description: "Default action for export route filter.",
+				Type:        schema.TypeString,
+			},
+			Attr_DefaultImportRouteFilter: {
+				Computed:    true,
+				Description: "Default action for import route filter.",
+				Type:        schema.TypeString,
+			},
+			Attr_Error: {
+				Computed:    true,
+				Description: "Error description.",
+				Type:        schema.TypeString,
+			},
+			Attr_ExportRouteFilters: {
+				Computed:    true,
+				Description: "List of export route filters.",
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						Attr_Action: {
+							Computed:    true,
+							Description: "Action of the filter.",
+							Type:        schema.TypeString,
+						},
+						Attr_CreationDate: {
+							Computed:    true,
+							Description: "Time stamp for create route filter.",
+							Type:        schema.TypeString,
+						},
+						Attr_Direction: {
+							Computed:    true,
+							Description: "Direction of the filter.",
+							Type:        schema.TypeString,
+						},
+						Attr_Error: {
+							Computed:    true,
+							Description: "Error description.",
+							Type:        schema.TypeString,
+						},
+						Attr_GE: {
+							Computed:    true,
+							Description: "The minimum matching length of the prefix-set.",
+							Type:        schema.TypeInt,
+						},
+						Attr_Index: {
+							Computed:    true,
+							Description: "Priority or order of the filter.",
+							Type:        schema.TypeInt,
+						},
+						Attr_LE: {
+							Computed:    true,
+							Description: "The maximum matching length of the prefix-set.",
+							Type:        schema.TypeInt,
+						},
+						Attr_Prefix: {
+							Computed:    true,
+							Description: "IP prefix representing an address and mask length of the prefix-set.",
+							Type:        schema.TypeString,
+						},
+						Attr_RouteFilterID: {
+							Computed:    true,
+							Description: "Route filter ID.",
+							Type:        schema.TypeString,
+						},
+						Attr_State: {
+							Computed:    true,
+							Description: "Status of the route filter.",
+							Type:        schema.TypeString,
+						},
+					},
+				},
+				Type: schema.TypeList,
+			},
+			Attr_IBMASN: {
+				Computed:    true,
+				Description: "ASN number at IBM PowerVS side.",
+				Type:        schema.TypeInt,
+			},
+			Attr_IBMCIDR: {
+				Computed:    true,
+				Description: "IP address used for configuring IBM network interface with network subnet mask.",
+				Type:        schema.TypeString,
+			},
+			Attr_ImportRouteFilters: {
+				Computed:    true,
+				Description: "List of import route filters.",
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
```

Comment:
Looks like the structure is the same as `Attr_ExportRouteFilters`, isn't it?
If yes, then a variable can be used to remove duplicate code.